buildenv/jenkins/testJobTemplate

```diff
@@ -326,8 +326,14 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							    sectionHeaderStyle(sectionHeaderHelpTextStyleCss)
 							}
 							choiceParam('TEST_FLAG', ['', 'JITAAS', 'AOT'], "Optional. Only set to JITAAS/AOT for JITAAS/AOT feature testing.")
-							stringParam('EXTRA_OPTIONS', EXTRA_OPTIONS, "Use this to append options to the test command")
-							stringParam('JVM_OPTIONS', "", "Use this to replace the test original command line options")
+							stringParam('EXTRA_OPTIONS', EXTRA_OPTIONS, "Use this to append options to the commandline",'''
+							Example :<br>
+							\'''-Xjit:{*LoaderThread.run{}V}{very-hot}\(traceCG,log=methodLog\)'\'''<br>
```

Comment:
I do not think `'''` is correct. https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/482/
I tried double quotes and it did not work. And I also tried the example in  https://github.com/adoptium/aqa-systemtest/issues/188. @Mesbah-Alam any idea why it does not work? https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/480/